### PR TITLE
I1302: remove a non-useful function (created by xiang in 0.0.8); add year of introduction

### PR DESCRIPTION
**Pull request by xiangli313**
* Youtrack ticket:
https://vimc.myjetbrains.com/youtrack/issue/VIMC-1302

* Task
In 0.0.8, I created a function to deal with SDF7 PSU Measles impact. Given that MR_Measles migration is (will be) approved at https://github.com/vimc/montagu-data/pull/34. This function is no longer needed.

In addition, founders want to have "year_of_introduction" in summary output files. I created a function to do the job. This function does not touch on existing functions for the modup. 

* Tested on science
No, I can only test jenner with orderly reports locally. Looks good to me.

### CODE REVIEW

File: R/modified_update.R [hunk 1]
```diff
@@ -537,105 +536,62 @@ mu_calculate_rate <- function(name, dat, window, n_years) {
...
+
+  a$year_intro <- NA
+  i <- !is.na(a$coverage) & a$coverage != 0.
+  if(any(i)){
+    a$year_intro <- min(a$year[i], na.rm = TRUE)
+  }
+  a <- unique(a[cols2])
+}
+
+find_year_introduction <- function(con, touchstone){
```
Comment (richfitz) [inline R/modified_update.R]: consider doing more in SQL

```r
d3 <- DBI::dbGetQuery(con, "SELECT DISTINCT
                               activity_type, vaccine, country, MIN(year) AS year_intro
                               FROM coverage
                               JOIN coverage_set
                               ON coverage_set.id = coverage.coverage_set
                               WHERE touchstone = $1
                               AND activity_type = 'routine'
                               AND coverage IS NOT NULL
                               AND coverage != 0
                               GROUP BY vaccine, country, activity_type
                               ORDER BY vaccine, country",
                     touchstone)
```

I think this is about right but might differ.  Please check.  With this you will no longer need the `year_intro` function , nor the split step below

Comment (richfitz) [reply R/modified_update.R]: Some docs: https://www.techonthenet.com/sql/min.php

Comment (richfitz) [reply R/modified_update.R]: I'm not 100% sure if the `DISTINCT` is needed once the `GROUP BY` is used

Comment (xiangli313) [reply R/modified_update.R]: Verified. It is supper nice. `DISTINCT` is not needed.

**Pull request merged**